src/buildSummary.ts

```diff
@@ -37,16 +21,31 @@ export function writeSummary(taskSummaryTableRows: string[][]) {
     }
 }
 
+export function getSummaryRows(buildSummary: string): any[] {
+    const rows = JSON.parse(buildSummary).map((t: { name: any; failed: { toString: () => any; }; skipped: { toString: () => any; }; description: any; duration: { toString: () => any; }; }) => {
```

Comment:
Is the type annotation needed here? If we can get away with omitting it I think that might be more readable in this case.

Reply:
it seems atleast one annotation should be there. else need to create a interface. i have changed the entire line to just `(t: any)`